README

```diff
@@ -173,17 +173,7 @@ We don't have a good way to test these.latter scripts.
 
 TG FeedController bug
 ---------------------
-
-We're using an old version of TG (1.0) that no longer gets any updates so
-we'll have to modify the local TG everytime.
-
-Apply this patch to turbogears/feeds/feed.py
-http://trac.turbogears.org/attachment/ticket/2348/feedkid.diff
-
-And here's the link to the upstream ticket:
-http://trac.turbogears.org/ticket/2348
-
-Note: this has been applied to the Fedora and EPEL packages.
+## Per the linked ticket, it appears this section is now  useless and outdated (current  Fedora / EPEL installs appear to bring in 1.1.x)
```

Comment:
This line is wider than the rest of the files so I guess it should be splt over two lines, no?